cocos/ui/UIRichText.h

```diff
@@ -116,7 +116,8 @@ class CC_GUI_DLL RichElementText : public RichElement
         URL_FLAG = 1 << 4,              /*!< url of anchor */
         OUTLINE_FLAG = 1 << 5,          /*!< outline effect */
         SHADOW_FLAG = 1 << 6,           /*!< shadow effect */
-        GLOW_FLAG = 1 << 7              /*!< glow effect */
+        GLOW_FLAG = 1 << 7,             /*!< glow effect */
+        TRIM_TRAILING_WHITESPACE = 1 << 8       /*!< trim right side spaces */
```

Comment:
Yep, I'll add what you requested.  I've never actually used the string keys, since I create the UIRichText in C++ code only, so for example:

                auto re1 = ui::RichElementText::create(1, color, 255, str, "fonts/Arimo-Regular.ttf", fontSize, ui::RichElementText::SHADOW_FLAG | ui::RichElementText::TRIM_TRAILING_SPACE);

Anyhow, I'll update the pull request with the changes you mentioned.